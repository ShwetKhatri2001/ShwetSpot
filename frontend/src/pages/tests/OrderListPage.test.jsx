import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';

import OrderListPage from '../OrderListPage';
import store from '../../redux/store';

describe('OrderListPage Component', () => {
  const MockOrderListPage = (
    <MemoryRouter>
      <Provider store={store}>
        <OrderListPage />
      </Provider>
    </MemoryRouter>
  );

  it('should match the snapshot', () => {
    const tree = renderer.create(MockOrderListPage).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
