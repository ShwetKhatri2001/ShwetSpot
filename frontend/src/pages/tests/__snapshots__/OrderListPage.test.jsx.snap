// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderListPage Component should match the snapshot 1`] = `
Array [
  <h1>
    Orders
  </h1>,
  <div
    className="table-responsive"
  >
    <table
      className="table-sm table table-striped table-bordered table-hover"
    >
      <thead>
        <tr>
          <th>
            ID
          </th>
          <th>
            USER
          </th>
          <th>
            DATE
          </th>
          <th>
            TOTAL
          </th>
          <th>
            PAID
          </th>
          <th>
            DELIVERED
          </th>
          <th />
        </tr>
      </thead>
      <tbody />
    </table>
  </div>,
]
`;
